src/momento-signer.ts

```diff
@@ -0,0 +1,213 @@
+import {InvalidArgumentError} from '.';
+import {importJWK, JWK, JWTHeaderParameters, KeyLike, SignJWT} from 'jose';
+
+export enum CacheOperation {
+  Get = 'GET',
+  Set = 'SET',
+}
+
+export interface signingRequest {
```

Comment:
I am not very sure what style guide we use but is `lowerCamelCase` the right way to name interfaces. We do `UpperCamelCase` for classes. Just seems a bit inconsistent.

Reply:
Oh good call... I think you're probably right that `SigningRequest` would be better here and same for `PresignedUrlRequest` below.

Reply:
Done in 771fd7f